packages/python-sqlalchemy/superfunctions_sqlalchemy/adapter.py

```
@@ -0,0 +1,571 @@
+
+    async def _delete(self, params: DeleteParams, connection=None) -> None:
+        """Delete a single record."""
+        try:
+            table = self._get_table(params.model, params.namespace)
+            where_clause = self._build_where_clause(table, params.where)
+
+            conn = connection
+            if conn is not None:
+                result = conn.execute(delete(table).where(where_clause))
+                if result.rowcount == 0:
+                    raise NotFoundError(f"Record not found for deletion")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)